cfg/rippled-example.cfg

```diff
@@ -1629,10 +1629,10 @@ ip = 127.0.0.1
 admin = 127.0.0.1
 protocol = ws
 
-#[port_grpc]
-#port = 50051
-#ip = 0.0.0.0
-#secure_gateway = 127.0.0.1
+[port_grpc]
```

Comment:
Why are we uncommenting this in the `rippled-example.cfg`. Do we want gRPC to be enabled by default?

Reply:
We talked about this in the huddle. I asked if we could uncomment this by default, and the agreement was yes if the xpring sdk code was removed.